README.md

```diff
@@ -32,6 +32,10 @@ Your application will be issued a client ID and client secret, which need to be
 provided to the strategy.  You will also need to configure a redirect URI which
 matches the route in your application.
 
+#### Enable Google Plus API
+
+Go to: [Google Plus API page in Developers Console](https://console.developers.google.com/apis/api/plus.googleapis.com/overview) and click on the Enable button next to the API name. Without it the redirect inside your Feathers app will result in Internal Server Error.
```

Comment:
My guess would be: https://feathersjs.com

Maybe that part was copied from the Feathers documentation?